ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -294,6 +296,36 @@ sub init_globals {
     return \%globals;
 }
 
+sub process_options {
+    my ($globals, $opts) = @_;
+
+    @{$globals->{Podpath}}  = split(":", $opts->{podpath}) if defined $opts->{podpath};
+
+    $globals->{Backlink}  =          $opts->{backlink}   if defined $opts->{backlink};
+    $globals->{Cachedir}  =  unixify($opts->{cachedir})  if defined $opts->{cachedir};
+    $globals->{Css}       =          $opts->{css}        if defined $opts->{css};
+    $globals->{Header}    =          $opts->{header}     if defined $opts->{header};
+    $globals->{Htmldir}   =  unixify($opts->{htmldir})   if defined $opts->{htmldir};
+    $globals->{Htmlroot}  =  unixify($opts->{htmlroot})  if defined $opts->{htmlroot};
+    $globals->{Doindex}   =          $opts->{index}      if defined $opts->{index};
+    $globals->{Podfile}   =  unixify($opts->{infile})    if defined $opts->{infile};
+    $globals->{Htmlfile}  =  unixify($opts->{outfile})   if defined $opts->{outfile};
+    $globals->{Poderrors} =          $opts->{poderrors}  if defined $opts->{poderrors};
+    $globals->{Podroot}   =  unixify($opts->{podroot})   if defined $opts->{podroot};
+    $globals->{Quiet}     =          $opts->{quiet}      if defined $opts->{quiet};
+    $globals->{Recurse}   =          $opts->{recurse}    if defined $opts->{recurse};
+    $globals->{Title}     =          $opts->{title}      if defined $opts->{title};
+    $globals->{Verbose}   =          $opts->{verbose}    if defined $opts->{verbose};
+
+    warn "Flushing directory caches\n"
+        if $opts->{verbose} && defined $opts->{flush};
```

Comment:
If you move this `warn` in side the conditional two lines down, you can simplify its trailing `if...`

Reply:
> If you move this `warn` in side the conditional two lines down, you can simplify its trailing `if...`

Again, this is a place where I wanted to avoid doing too much refactoring too soon.  So I retained the flow of the code in the version which shipped with perl-5.34:
```
572     warn "Flushing directory caches\n"
573         if $opt_verbose && defined $opt_flush;
574     $Dircache = "$Cachedir/pod2htmd.tmp";
575     if (defined $opt_flush) {
576         1 while unlink($Dircache);
577     }
```
So, I'd like to defer this until after we've got our Pod::Html object.  I'll add it to the [pod-html TODO list](http://thenceforward.net/perl/misc/pod-html-remaining-issues.txt).